ft-sdk/src/query.rs

```diff
@@ -1,8 +1,20 @@
+/// Store query variables
+///
+/// for query string like `?name=foo&age=20`, it will be stored as
+/// `vec![("name", "foo"), ("age", "20")]`
```

Comment:
Should we document this internal/non public detail?

Reply:
you're right. We only access these through the method `get` so it does not make sense to have this piece of documentation. Fixed in https://github.com/FifthTry/ft-sdk/pull/5/commits/4039918eff526b544529dbe7c7d2621f05e4f05a